packages/strapi-design-system/src/Layout/HeaderLayout.js

```diff
@@ -137,7 +147,7 @@ BaseHeaderLayout.propTypes = {
   primaryAction: PropTypes.node,
   secondaryAction: PropTypes.node,
   sticky: PropTypes.bool,
-  subtitle: PropTypes.string,
+  subtitle: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
Could you add a comment:

```
// TODO V2: Remove the string fallback
```

to make sure we remove that in the next major iteration?